editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -502,6 +464,9 @@ void CanvasItemEditor::_find_canvas_items_at_pos(const Point2 &p_pos, Node *p_no
 			CanvasLayer *cl = p_node->cast_to<CanvasLayer>();
 			_find_canvas_items_at_pos(p_pos, p_node->get_child(i), transform, cl ? cl->get_transform() : p_canvas_xform, r_items); //use base transform
 		}
+
+		if (limit && r_items.size() >= limit)
```

Comment:
For me, if (limit) suggests limit is a pointer and then an * is missing from the other part of the condition. Checking for 0 without comparison is done in Godot for size() and stuff like that, but for a variable it's misleading.